MD/md-n/README.md

```diff
@@ -0,0 +1,137 @@
+
+[PBFT](https://pmg.csail.mit.edu/papers/osdi99.pdf) introduced a protocol which ensures consecutive rounds of consensus intersect in at least one unit of honest voting power via the Generalized Pigeon Hole Principle. The transitivity of this property ensures that a chain of consensus rounds must intersect in at least one unit of honest voting power. 
+
+Under an [experts model](https://people.csail.mit.edu/ghaffari/AA19/AAscript.pdf?utm_source=chatgpt.com) understanding of PBFT systems, we can make an assumption that there is a perfect expert and thus that this perfect expert is identified by consensus. This, in turn, renders PBFT a lossless approximate algorithm.
+
+If we consider allowing forks, however, it is initially unclear how perfectness translates. Naively, our assumption that there is globally one perfect expert means that there is one **perfect** fork. 
+
+We can, however, apply a reduced criterion and state that a each **fork** be comprised of a chain of consensus rounds each of which intersect in one unit of honest voting power w.r.t. to the fork itself. We call this **fork-perfectness**.
+
+Preserving **fork-perfectness** reduces to ensuring the fraction of stake which decided the original fork maintains a supermajority. If this does not occur, then it is impossible to have the guaranteed intersection. 
+
+### A3: Awareness of Minority Decisions
+
+The requirement of **full-synchronicity** means that if a supermajority decision is not made by Global Stabilization Time, some form of minority decision must effectively be made. This does not inherently mean that a Byzantine fraction of voting power may decide the global state. For example, if long-lived forks are preserved, there are multiple global states and eventual supermajority consistency can be achieved via a reconstitution of supermajority stake on an honest fork. In other words, while a **partially-synchronous** system renders eventual liveness and guaranteed safety, a **full-synchronous** system can render guaranteed liveness and eventual safety. 
+
+Regardless, Dongmen Postconfirmation protocols are required to provide a formal model of the expected value of consensus on a Byzantine minority w.r.t. the value of the state and intrinsic rewards. In other words, they must explain the nature of eventual safety that abides by Byzantine assumptions or else describe the non-BFT nature of the protocol at full-synchronous decision points. 
+
+### A4: Example Minority-Aware Protocol
+Consider the following fully-synchronous protocol:
+
+1. Votes $v \in V$ are cast for states $s \in S$ at height $h \in H$.
+2. A decision in made on a vote $s, h$ by time $t \in T$.
+    1. If $V_{h}(s) > \frac{2*|V|}{3}$, accept $s, h$
+    2. Otherwise, begin **Play Foward** algorithm.
+
+The **Play Forward** algorithm is as follows:
+
+Given a duration $d: (t + d) \in T$, and a slot duration $l: (d/l) \in T$, for each slot $l'$ in the duration $t, t + l, ... t + d$ $h' = h + (l'/l)$. 
```

Comment:
There are a few errors in that line
- $l$ is not a slot duration but a resolution of the duration $d$, wheras $d/l$ seems the slot duration
- the end of the sentence is not very readable, define h' not within the equation. 
- its confusing to define l' as duration, whereas you define l as a resolution factor. Use the same type of units (i.e. time) for them. indeed it doesnt seem necessary to define a resolution.
-  also something seems wrong about the end of the sentence 
- generally this sentence is hard to read, we are not only targeting math people

Reply:
```suggestion
Assume durations $d$ , and a slot duration $l$, where $l<d$. Assume the protocol progresses in steps of slot duration $l$, i.e. the following time steps exist: $t, t+l , t+2l, .. , t+l' \in T$ , where $l` = \delta h \cdot l$ and $h' = h + \delta h$
```